streamwebs_frontend/streamwebs/tests/models/test_water_quality.py

```diff
@@ -0,0 +1,300 @@
+from django.test import TestCase
+
+from django.contrib.gis.db import models
+from django.apps import apps
+from itertools import chain
+
+from streamwebs.models import Site
+from streamwebs.models import Water_Quality
+from streamwebs.models import WQ_Sample
+
+
+class WaterQualityTestCase(TestCase):
+
+    def setUp(self):
+        self.expected_fields = {
+            'site': models.ForeignKey,
+            'site_id': models.ForeignKey,
+            'DEQ_wq_level': models.CharField,
+            'date': models.DateField,
+            'school': models.CharField,
+            'latitude': models.DecimalField,
+            'longitude': models.DecimalField,
+            'fish_present': models.BooleanField,
+            'live_fish': models.PositiveSmallIntegerField,
+            'dead_fish': models.PositiveSmallIntegerField,
+            'water_temp_unit': models.CharField,
+            'air_temp_unit': models.CharField,
+            'sample_1': models.ForeignKey,
+            'sample_2': models.ForeignKey,
+            'sample_3': models.ForeignKey,
+            'sample_4': models.ForeignKey,
+            'notes': models.TextField,
+            'id': models.AutoField,
+
+            # Corresponding sample entry (id)
+            'sample_1_id': models.ForeignKey,
+            'sample_2_id': models.ForeignKey,
+            'sample_3_id': models.ForeignKey,
+            'sample_4_id': models.ForeignKey
+        }
+
+        self.optional_fields = {
+           'notes'
+        }
+
+    def test_fields_exist(self):
+        model = apps.get_model('streamwebs', 'water_quality')
+        for field, field_type in self.expected_fields.items():
+            self.assertEqual(
+                field_type, type(model._meta.get_field(field)))
+
+    def test_no_extra_fields(self):
```

Comment:
I like having docstrings for all of my test_* methods, so I can immediately see what the error is. Your call though, it's not a big deal.